### ISSUE TITLE
`#[options(name = ...)]` seems to be undocumented.

### ISSUE THREAD

**Issue by vi** [murarth/gumdrop#55]
[docs.rs page](https://docs.rs/gumdrop_derive/0.8.0/gumdrop_derive/) list possible variants for `#[options(...)]`, but I don't see `name` there.
It can be used to rename subcommands.

**Comment by murarth** [comment]
You're right. Thanks for noticing and reporting this. I've just pushed a commit to document the `#[options(...)]` attribute for `enum` types.
